config/initializers/rack_profiler.rb

```diff
@@ -0,0 +1,6 @@
+if Rails.env == 'development'
```

Comment:
I'd check if the profiler makes that big a difference; I'd guess compared to everything else argo does, probably not.